### PR TITLE
change test values for dashboard refresh interval

### PR DESCRIPTION
**Pull request by seansain**
```
=== RUN   TestAccSumologicDashboard_create
    testing.go:705: Step 0 error: errors during apply:
        
        Error: {"id":"MGKAJ-ERO35-M1O8H","errors":[{"code":"dashboard:invalid_refresh_interval","message":"Refresh interval 30 is invalid for dashboard FFFFFFFFFFFFFFFF. Please use from the following list: 120,300,900,1800,3600,7200,86400.","meta":{"refreshInterval":{"bytes":"MzA=","length":2,"byteLength":2},"dashboardId":{"bytes":"RkZGRkZGRkZGRkZGRkZGRg==","length":16,"byteLength":16}}}]}
        
          on /tmp/tf-test106556792/main.tf line 3:
          (source code not available)
        
        
--- FAIL: TestAccSumologicDashboard_create (1.31s)
=== RUN   TestAccSumologicDashboard_update
    testing.go:705: Step 0 error: errors during apply:
        
        Error: {"id":"BULW1-323FD-X6IQJ","errors":[{"code":"dashboard:invalid_refresh_interval","message":"Refresh interval 30 is invalid for dashboard FFFFFFFFFFFFFFFF. Please use from the following list: 120,300,900,1800,3600,7200,86400.","meta":{"refreshInterval":{"bytes":"MzA=","length":2,"byteLength":2},"dashboardId":{"bytes":"RkZGRkZGRkZGRkZGRkZGRg==","length":16,"byteLength":16}}}]}
        
          on /tmp/tf-test920048490/main.tf line 3:
          (source code not available)
```

### CODE REVIEW

File: sumologic/resource_sumologic_dashboard_test.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func TestAccSumologicDashboard_create(t *testing.T) {
 	title := "terraform_test_dashboard_" + testNameSuffix
 	description := "Test dashboard description"
 	theme := "Dark"
-	refreshInterval := 30
```
Comment (sumovishal) [inline sumologic/resource_sumologic_dashboard_test.go:43]: Why is 30 an invalid value for `refreshInterval`?

Comment (seansain) [reply sumologic/resource_sumologic_dashboard_test.go:43]: Not sure, that's just what the error message says.

Comment (seansain) [reply sumologic/resource_sumologic_dashboard_test.go:43]: `Please use from the following list: 120,300,900,1800,3600,7200,86400.`
Is this a breaking change?

Comment (sumovishal) [reply sumologic/resource_sumologic_dashboard_test.go:43]: Yeah, if we only allow certain values, this is a breaking API change.

File: website/docs/r/dashboard.html.markdown [hunk 2]
```diff
@@ -272,7 +272,7 @@ The following arguments are supported:
 - `description` - (Optional) Description of the dashboard.
 - `folder_id` - (Optional) The identifier of the folder to save the dashboard in. By default it is saved in your
 personal folder.
-- `refresh_interval` - (Optional) Interval of time (in seconds) to automatically refresh the dashboard.
+- `refresh_interval` - (Optional) Interval of time (in seconds) to automatically refresh the dashboard. Valid values are 120,300,900,1800,3600,7200,86400.
```
Comment (sumovishal) [inline website/docs/r/dashboard.html.markdown:275]: We should add the validation in the resource as well.

**Review by sumovishal — APPROVED** [review]
lgtm, except one minor issue.

File: website/docs/r/dashboard.html.markdown (hunk 2 above)
Comment (seansain) [reply website/docs/r/dashboard.html.markdown:275]: Ohh ya, I'll add that

**Pull request merged**